docker/fedora-41-nightly.Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN dnf install -y \
 # Bust the cache
 ARG STAMP=1729535688
 
-RUN dnf config-manager --add-repo https://download.opensuse.org/repositories/security:zeek/Fedora_39/security:zeek.repo
+RUN dnf config-manager addrepo --from-repofile=https://download.opensuse.org/repositories/security:zeek/Fedora_40/security:zeek.repo
```

Comment:
Using packages for fedora-40 on fedora-41 looks like a typo but is actually deliberate (no upstream packages for that distro yet). Could you add a comment calling that out? Adding support for fedora-41 upstream would also be great, maybe you could reference an issue.